src/command.rs

```diff
@@ -214,8 +235,21 @@ impl Command {
         Ok(self)
     }
 
+    fn _console(&self) -> Result<&Self, Error> {
+        trace!("calling _console and passing to shell");
+
+        if Path::new("pndev.toml").exists() {
+            shell::run("bundle exec rails console")?;
```

Comment:
it might be better to parse this OUT of the pndev.toml file 

add a `console =` line there and have that define how to run the console? similar to how `pndev start` works

Reply:
Agreed. Imagine we had a hypothetical Elixir project. I'd expect this to do `iex -S mix`

Reply:
Going a step further, it might be nice to support arbitrary command in general. Imagine...

```
$ pndev run console
# start some kind of repl

$ pndev run db
# could proxy to bin/rails db -p

$ pndev run lint
```

You get the idea.

Reply:
oh this is a good idea. maybe we can pull out the console one from this pr @typeoneerror and rewrite it as `pndev run XXX`
and XXX is looked up in `pndev.toml` and ran inside `shell::run(`

Reply:
@jherdman great idea. we just discussed this and should be not too much trouble to implement. will get back on this on Friday.